### PR TITLE
localization Validation update

### PR DESCRIPTION
**Pull request by PreethiMaai**

### CODE REVIEW

File: cypress/support/validations/decodeValidation.js [hunk 1]
```diff
@@ -130,10 +130,9 @@ class decodeValidations {
 
         if (decode.includes(param)) {
           const indexOfParam = decode.indexOf(param);
-          const extractedData = decode.slice(
-            indexOfParam + param.length + 2,
-            decode.indexOf('</', indexOfParam + 1)
```
Comment (neeradanelxsi) [inline cypress/support/validations/decodeValidation.js:135]: will this fail rajankikas changes ? If this was for advertising fix, we have another ticket

File: cypress/fixtures/fireboltCalls/localization.json [hunk 2]
```diff
@@ -102,7 +102,7 @@
     "PH_FOR_LOCALIZATION_COUNTRYCODE": {
         "method": "localization.countryCode",  
         "validationJsonPath": "result",
-        "content": "LOCALIZATION_COUNTRYCODE"  
+        "content": "LOCALIZATION_COUNTRYCODE_PH"  
     },
```
Comment (neeradanelxsi) [inline cypress/fixtures/fireboltCalls/localization.json:106]: dont understand the change here. Which test case will  be fixed with this change

Comment (PreethiMaai) [reply cypress/fixtures/fireboltCalls/localization.json:106]: Updated localization regEx validation to content validation. So, both method and event have same validation.
Reports:
Localization: [Multiple Cucumber HTML Reporter (comcast.com)](https://reporting.firecert.comcast.com/cucumberReports/dfb3b774-4121-4c31-b5fd-cf7c92101a0c/index.html)
LocalizationManage: [Multiple Cucumber HTML Reporter (comcast.com)](https://reporting.firecert.comcast.com/cucumberReports/59cfe590-e588-45dc-815e-295206326e23/index.html)

**Review by ibrahimshuail — APPROVED** [review]

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**